go/src/koding/klientctl/auth.go

```diff
@@ -92,6 +92,8 @@ func AuthLogin(c *cli.Context, log logging.Logger, _ string) (int, error) {
 		}
 	}
 
+	fmt.Fprintln(os.Stderr, "Logging to", kodingURL, "...")
```

Comment:
I don't share this opinion, I took an approach that a final message that is the result of the command get printed to os.Stdout, anything that is printed before are just logs and printed to os.Stderr.

Without this assumption it won't be possible to write a test.